README.md

```diff
@@ -0,0 +1,62 @@
+![logo](https://upload.wikimedia.org/wikipedia/commons/b/b5/Kore_2g_logo.png)
+
+* OpenKore is an custom client and intelligent automated assistant for Ragnarok Online.
+* It is a **free**, open source and cross-platform program (Linux and Windows are supported).
+
+## Prerequisites
+
+To run OpenKore you will need:
+1. [Download and Install Perl](http://openkore.com/index.php/How_to_run_OpenKore#Perl) (You can run start.exe or wxstart.exe if you use microsoft windows)
```

Comment:
Perhaps this could be one sentence instead of numbered list with just one item since it seems that Perl is the only dependency/prerequisite to run OK.

Actually, why not to link http://wiki.openkore.com/index.php/How_to_run_OpenKore#Requirements here?

Reply:
@zstyblik  
[Download and Install Perl] > [How to run OpenKore](http://wiki.openkore.com/index.php/How_to_run_OpenKore#How_to_run_OpenKore)
(You can run start.exe or wxstart.exe if you use microsoft windows)


What do you think ?

Reply:
@sctnightcore, how about: 

> To run OpenKore you have to download and install Perl. Please, see [How to run OpenKore](http://wiki.openkore.com/index.php/How_to_run_OpenKore#How_to_run_OpenKore) for more information. Once you have Perl installed, you can run `start.exe` or `wxstart.exe`, if you use Microsoft Windows.

I still feel it's a bit misleading, because it seems to me there are some additional Perl modules that must be installed via `ppm` on Windows, but I cannot say whether it's hard requirement or not as I run OK at Linux.

PS: I'm not a native English speaker.